yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerSSHLauncher.java

```diff
@@ -63,18 +63,23 @@ public ComputerLauncher getPreparedLauncher(String cloudId, DockerSlaveTemplate
     }
 
     @Override
-    public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, CreateContainerCmd createCmd) {
-        final int sshPort = getSshConnector().port;
-
-        createCmd.withPortSpecs(sshPort + "/tcp");
-
-        String[] cmd = dockerSlaveTemplate.getDockerContainerLifecycle().getCreateContainer().getDockerCommandArray();
-        if (cmd.length == 0) {
-            //default value to preserve compatibility
-            createCmd.withCmd("bash", "-c", "/usr/sbin/sshd -D -p " + sshPort);
+    public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, CreateContainerCmd createCmd, String osType) {
+        if (osType.equals("windows")) {
+            // TODO: figure out how to support SSH launcher on Windows
+            throw new RuntimeException("Windows OS is not supported for SSH launcher");
```

Comment:
In addition: I had it almost running with the win-openssl implementation which lacks a fix (dunno which exactly).